File: class_primer/class_primer__change_member.cs
using static System.Console;

internal class Program
{
    class Employee
    {
        int Num;
        string Name;

        internal Employee( int i, string s )
        {
            Num = i;
            Name = s;
        }

        internal void ChangeName( string s )
        {
            Name = s;
            return;
        }

        internal void ChangeNum( int i )
        {
            Num = i;
            return;
        }

        internal string GetName()
        {
            return Name;
        }

        internal int GetNum()
        {
            return Num;
        }
    }

    static void Main()
    {
        int n = int.Parse( ReadLine() );
        var l = new System.Collections.Generic.List<Employee>( n );

        for( int _ = 0; _ < n; _++ ) {
            string[] s = ReadLine().Split();

            switch( s[0] ) {
            case "make":
                l.Add( new Employee( int.Parse( s[1] ), s[2] ) );
                break;
            case "getnum":
                WriteLine( l[int.Parse( s[1] ) - 1].GetNum() );
                break;
            case "getname":
                WriteLine( l[int.Parse( s[1] ) - 1].GetName() );
                break;
            case "change_num":
                l[int.Parse( s[1] ) - 1].ChangeNum( int.Parse( s[2] ) );
                break;
            case "change_name":
                l[int.Parse( s[1] ) - 1].ChangeName( s[2] );
                break;
            }
        }
        return;
    }
}

File: class_primer/class_primer__change_member.py
class employee:
    def __init__(self, num, name):
        self.num = num
        self.name = name

    def change_name(self, name):
        self.name = name

    def change_num(self, num):
        self.num = num

    def getname(self):
        return self.name

    def getnum(self):
        return self.num

n = int(input())
l = []

for _ in range(n):
    s = input().split()
    i = int(s[1])

    if s[0] == "make":
        l.append(employee(i, s[2]))
    elif s[0] == "getnum":
        print(employee.getnum(l[i - 1]))
    elif s[0] == "getname":
        print(employee.getname(l[i - 1]))
    elif s[0] == "change_num":
        employee.change_num(l[i - 1], int(s[2]))
    elif s[0] == "change_name":
        employee.change_name(l[i - 1], s[2])
